internal/exports/stdout_writer.go

```diff
@@ -12,17 +12,21 @@ func StdoutCallbackFactory(
 	out io.Writer,
 	writeTimeseries bool,
 ) (common.Callback, common.CallbackTeardown) {
+	errorStats := common.NewErrorStats()
 
 	return func(pkg common.DataRecord) {
-		if writeTimeseries {
-			if pkg.Error != nil {
-				pkg.Error = fmt.Errorf(
-					"%s %s",
-					pkg.Error,
-					common.MakePackageInfo(&pkg),
-				)
+			errorStats.Register(pkg.Error)

```

Comment:
Looks like the indentation is off here, which is weird since that is not under your control, right? Please have a look at it nonetheless.

Reply:
no go decides indentations and this is as it should be, though it looks dead ugly to me.